pages/terraform-provider.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <div class="main">
+        <div class="container">
+            <TerraformHeader />
+            <TerraformOverview />
+            <TerraformUseCases />
+            <TerraformFooter />
+        </div>
+    </div>
+</template>
+
+<style scoped lang="scss">
+.main {
+    background-color: #0F011D;
```

Comment:
Add a variable, create one if necessary, so we can list all colors used in website